### DISCUSSION TITLE
Unable to render data after Build

### DISCUSSION THREAD

**Discussion by Srikant1290** [lumada-design/hv-uikit-react#4162] [Q&A]
<HvTableRow {...row.getRowProps()}>
        {row.cells.map((cell) => (
          <HvTableCell {...cell.getCellProps()}>
            {cell.render("Cell")}
          </HvTableCell>
        ))}
      </HvTableRow>
      
      I am not able to see the data in the dom itself after Build

**Comment by zettca** [comment]
Issue: https://github.com/lumada-design/hv-uikit-react/issues/4163
